.github/workflows/release.yml

```diff
@@ -0,0 +1,134 @@
+name: "Release New Version"
+run-name: "Release ${{ inputs.version }}"
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "The version to be released. This is checked for consistency with the branch name and configuration"
+        required: true
+        type: "string"
+      jira-version-number:
+        description: "JIRA version ID (e.g. 54321)"
+        required: true
+        type: "string"
+
+env:
+  # TODO: Use different token
+  GH_TOKEN: ${{ secrets.MERGE_UP_TOKEN }}
```

Comment:
I noticed this TODO is still present in PHPC (https://github.com/mongodb/mongo-php-driver/blob/0ced119f9d95da5d1c1a06fe412f9d26bce982e8/.github/workflows/release.yml#L18). Should there be a JIRA ticket to track this?

Reply:
The idea is to use a token for a GitHub app, which makes it easier to allow bypassing branch protection rules. Tracking in PHPC-2383